docs/releases/1.26-NOTES.md

```diff
@@ -8,6 +8,8 @@ This is a document to gather the release notes prior to the release.
 
 * The default networking provider for new clusters is now Cilium.
 
+* AWS Load Balancer Controller will now run on worker nodes if IRSA is enabled and the Kubernetes version is greater than 1.24.
```

Comment:
This should be in the AWS Only subsection

Reply:
Greater than or equal to 1.24

Reply:
This can be misconstrued as stating that LBC will be automatically enabled when k8s is new enough and IRSA is enabled. I suggest some sort of "if it is enabled" wording.